frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/PortalConstants.java

```diff
@@ -366,6 +366,7 @@
     public static final String DATATABLE_DISPLAY_DATA = "aaData";
     public static final String DATATABLE_RECORDS_TOTAL = "recordsTotal";
     public static final String DATATABLE_RECORDS_FILTERED = "recordsFiltered";
+    public static final String UPDATE_ONLY_REQUESTED = "updateOnlyRequested";
```

Comment:
I will use this in integration part: this allow us to control if I need updates for all fields.

Reply:
OK, but then it is the question if this should  be part of this PR, because it would prevent cherry picking your next one containing the integration part.

Reply:
also this is flag/trigger to manipulate portion update of component